docs/v3-api.yaml

```
@@ -1086,14 +1086,37 @@ paths:
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/UserGroupMember'
+              oneOf:
+                - $ref: '#/components/schemas/UserGroupMember'
+                - $ref: '#/components/schemas/UserGroupMembers'
```

難しいところだけど、実装で[]PostUserGroupMemberRequestとしているし、以下の形式でもいい気がするんだけど、読みにくいから今の形でも良いと思う。@kaitoyama 意見があれば聞きたい～
```suggestion
                - $ref: '#/components/schemas/UserGroupMember'
                - type: array
                  items:
                    $ref: '#/components/schemas/UserGroupMember'
```

---

自分もどっちでもいい気はする。ただ、個人的には元の方が揃ってて綺麗な気がするのと、codegen系を使うなら元の方が色々都合がいいことがあった経験があるので、元の方が好みかな〜

---

同感です。これは元のままで大丈夫です～！